dashboard/app/controllers/programming_environments_controller.rb

```diff
@@ -79,7 +79,7 @@ def programming_environment_params
       :title,
       :published,
       :description,
-      :editor_type,
+      :editor_language,
```

Comment:
Is it okay to change this in the same PR as the config files? Or does this need to be a two step process? No users should be using these pages yet so there isn't a risk of user impact, just build impact

Reply:
After talking to Dave about this change, the riskiest part of this will be if the DTL fails and editing is happening while the failure is just sitting there. As we DTL in the middle of the night, I'm going to merge this and do an out-of-bound DTL one evening so that I can address any issues that arise